doc/ongoing_operations.rst

```diff
@@ -40,14 +40,33 @@ The sync to the ``main`` repository affects all architectures.
 Guidelines for gating a sync to main
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When preparing for a sync it is recommended to review the
-*release status page*.
-There are filters for viewing regressions, and what packages will sync if the
-sync to main is run as well as the ability to search.
-
-Note: A version downgrade is also marked as a regression since if users have
-already installed the previous package with the higher version number apt will
-not install the newer package with a lower version automatically.
+There is a manual process to go from testing to main.
+We refer to this as a sync.
+
+When you are preparing for a sync:
+
+* Run the ``rosdistro_audit.py`` script it will output text which is appropriate for posting to Discourse as a triage list.
```

Comment:
```suggestion
* Run the ``rosdistro_audit.py`` script.
  It will output text which is appropriate for posting to Discourse as a triage list.
```
or

```suggestion
* Run the ``rosdistro_audit.py`` script which will output text which is appropriate for posting to Discourse as a triage list.
```

I like the first suggestion as it's simpler English but either would do.